std/traits.d

```diff
@@ -5074,6 +5075,31 @@ template BuiltinTypeOf(T)
     else                                        static assert(0);
 }
 
+/++
+    Helper template so that `typeof can be used with templates taking other
+    templates (such as $(REF, staticMap, std, meta)).
+  +/
+template TypeOf(alias exp)
```

Comment:
Have `TypeOf` work on types even though `typeof` doesn't work on types? As I said, I don't think that it makes any sense to get the type of a type, and I'm not going to make that change.